apps/prairielearn/src/lib/preact-html.ts

```diff
@@ -0,0 +1,12 @@
+import { render } from 'preact-render-to-string/jsx';
+
+import { type HtmlSafeString, unsafeHtml } from '@prairielearn/html';
+import type { VNode } from '@prairielearn/preact-cjs';
+
+// This function must live outside of `preact.tsx` so that it can be imported
+// on the client. `preact.tsx` imports `@prairielearn/compiled-assets`, which
+// cannot be bundled for the browser.
+
+export function renderHtml(vnode: VNode): HtmlSafeString {
+  return unsafeHtml(render(vnode, {}, { pretty: true, jsx: false }));
```

Comment:
I _think_ `jsx: false` is the default, maybe we can remove that? Though I see that I was importing from `preact-render-to-string/jsx`, and I don't remember why that was.

Reply:
I don't quite understand why you would want to generate jsx output. For the `preact-render-to-string/jsx` import, JSX is the default. For `preact-render-to-string`, JSX is not the default.